.github/workflows/build-and-test.yml

```diff
@@ -51,10 +51,31 @@ jobs:
           else
             echo "pr-number=${{ github.event.inputs.pr-number }}" >> $GITHUB_OUTPUT
           fi
-  
+
       - name: Build Docker image
         run: docker build -t ${{ vars.ARTIFACTORY_URL }}/${{ secrets.ARTIFACTORY_REPOSITORY_NAME }}/house-innovations-housnav-web:${{ steps.set-pr-number.outputs.pr-number }} -f apps/web/Dockerfile . --no-cache
 
+      - name: Run Docker container
+        run: docker run -d -p 3000:3000 --name housnav ${{ vars.ARTIFACTORY_URL }}/${{ secrets.ARTIFACTORY_REPOSITORY_NAME }}/house-innovations-housnav-web:${{ steps.set-pr-number.outputs.pr-number }}
+
+      - name: Install Cypress dependencies
+        run: npm install cypress
```

Comment:
Are we worried about possible cypress upgrades here?

Reply:
Changed to `npm ci`

Reply:
Do we care that that will cause us to install all the dependancies instead of just cypress?

Reply:
Looks like we can't choose the specific package to install from package-lock.json. We have to choose either to install all packages with `npm ci` or to specify the Cypress version in CI with `npm install cypress@13.9.0`.